src/stories/Quiz/Component/ComponentStories.stories.tsx

```diff
@@ -68,39 +70,34 @@ function RenderInASmallContainerTemplate(args: IQuizProps) {
     </div>
   );
 }
-export const RenderInASmallContainer = RenderInASmallContainerTemplate.bind({});
+
+export const RenderInASmallContainer: Story = {
+  args: {},
+  render: (args) => <RenderInASmallContainerTemplate {...args} />,
+};
 
 RenderInASmallContainer.args = { apiKey, quizId, callbacks, resultCardOptions };
 
 const cioJsClient = new ConstructorIOClient({ apiKey });
 
-const cioClientStoryResultsPageOptions = {
-  onAddToCartClick: callbacks.onAddToCartClick,
-  resultCardRegularPriceKey: resultCardOptions.resultCardRegularPriceKey,
-};
-
-export const ProvideCIOClientInstance = ComponentTemplate.bind({});
-
-ProvideCIOClientInstance.args = {
-  quizId,
-  resultsPageOptions: cioClientStoryResultsPageOptions,
-  cioJsClient,
-  resultCardOptions,
+export const ProvideCIOClientInstance: Story = {
+  args: {
+    quizId,
+    resultsPageOptions: {},
```

Comment:
Seems we've been passing wrong values before and just not catching it with ts? 🤔